docs/it-manual/configure-gis-service.md

```diff
@@ -1,51 +1,149 @@
-# How to Configure your GIS service for address correction and service area validation
+# How to configure your GIS service for address correction and service area validation
 
-There are several configuration values needed to enable address correction and service area validation. These features rely on an external Esri service and are not compatible with other geolocation services.
+You may have some programs that need to collect an applicant's address to offer
+services. Those programs may also want to verify that the address exists and
+matches a standard format, and may also want to verify that the address is in
+the area that's eligible for the program. For example, a program may only be for 
+applicants living in a certain county. CiviForm uses the external Esri service
+for both these use cases. CiviForm is not currently compatible with other
+geolocation services.
```

Comment:
**suggestion, non-blocking**: Should we add a note like "If you'd like to use another geolocation service, please reach out the the eng team" so it doesn't completely close the door to other services?

Reply:
Good idea, done.